### PR TITLE
Population done, mouse AUX buttons, Frontend, Hud, fixes

### PR DESCRIPTION
**Pull request by erorcun**

### CODE REVIEW

File: src/core/Frontend.cpp [hunk 1]
```diff
@@ -4012,6 +3717,10 @@ CMenuManager::ProcessButtonPresses(uint8 goDown, uint8 goUp, uint8 optionSelecte
 				MouseButtonJustClicked = 4;
 			else if (CPad::GetPad(0)->GetMouseWheelDownJustUp())
 				MouseButtonJustClicked = 5;
+			else if (CPad::GetPad(0)->GetMouseX1JustUp())
+				MouseButtonJustClicked = 6;
+			else if (CPad::GetPad(0)->GetMouseX2JustUp())
+				MouseButtonJustClicked = 7;
 			// TODO two more buttons
```
Comment (Nick007J) [inline src/core/Frontend.cpp:3724]: this todo is obsolete?

Comment (erorcun) [reply src/core/Frontend.cpp:3724]: ah yes

File: src/peds/Population.cpp [hunk 2]
```diff
@@ -580,49 +553,74 @@ CPopulation::AddToPopulation(float minDist, float maxDist, float minDistOffScree
...
+			for (i = 0; i < NUM_GANGS; i++) {
 				if (decisionThreshold < zoneInfo.gangPedThreshold[i]) {
-					pedTypeToAdd = PEDTYPE_GANG1 + i;
 					break;
 				}
+			}
 
-				if (i == NUM_GANGS - 1) {
+			if (i == NUM_GANGS) {
+				if (CGeneral::GetRandomNumberInRange(0, 1) <= 0.95f) {
```
Comment (Nick007J) [inline src/peds/Population.cpp:590]: 0.0f, 1.0f

**Review by Nick007J — APPROVED** [review]
lgtm except notes

**Comment by parkerlreed** [comment]
Does the miami branch not have arm64 configs? Was going to test the changes

**Comment by parkerlreed** [comment]
Ok now I'm just extra confused. Is miami a different game mode or just a name for the dev branch?

Both miami here and on the PR repo reference a file that's not in the Steam GTA3

```
[DBG]: Initialising CPedStats...
[DBG]: Loading pedstats data...
[DBG]: CPedStats ready
[DBG]: Intialising CTimeCycle...
[DBG]: CTimeCycle ready
[DEV]: release_existing()
[DEV]: set_new_provider()
[DEV]: EFX
casepath couldn't find dir/file "WILD.ADF", full path was AUDIO\WILD.ADF
[DEV]: Stream ./audio/WILD.ADF
Segmentation fault (core dumped)
```

**Comment by withmorten** [comment]
@parkerlreed miami is the branch for Vice City! It was added after 3 was done.

**Comment by parkerlreed** [comment]
Oh haha! Thanks. I feel stupid now.

**Review by ShFil119 — APPROVED** [review]
looks ok, but you need to rebase

**Pull request merged**